sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/CompilationExtensions.cs

```diff
@@ -29,5 +29,10 @@ public static class CompilationExtensions
             (IMethodSymbol)compilation.GetSpecialType(type)
                 .GetMembers(methodName)
                 .SingleOrDefault();
+
+        public static bool IsNetFrameworkTarget(this Compilation compilation) =>
+            // There's no direct way of checking current target framework yet (09/2020).
```

Comment:
I think the message is a bit confusing. We can check the target framework, by using precompiler symbols, but we cannot check the runtime.